Components/Transforms/DeformationFieldTransform/elxDeformationFieldTransform.hxx

```diff
@@ -159,9 +159,6 @@ DeformationFieldTransform<TElastix>::WriteToFile(const ParametersType & param) c
 
   typedef itk::ChangeInformationImageFilter<DeformationFieldType> ChangeInfoFilterType;
 
-  /** Add some DeformationFieldTransform specific lines. */
-  xout["transpar"] << std::endl << "// DeformationFieldTransform specific" << std::endl;
-
```

Comment:
Please check that `DeformationFieldTransform::WriteToFile` cannot be entirely removed!

Reply:
indeed

Reply:
@mstaring So is it OK to still have some parameters written to file that are not in the parameter map?

Reply:
Or would it be an option to add the parameter "DeformationFieldFileName" (which is now only written to file) to the map as well?

Reply:
I guess ideally you would push the whole deformation field to that map. Or at least the filename as you say, so that one can read it in again via disk. Lets discuss over sprint.

Reply:
By the way, the "making" of the DeformationFieldFileName throws an exception (`std::out_of_range`) inside `DeformationFieldTransform::WriteToFile` when the current TransformParameterFileName is not specified. Would it be OK to suppress (or avoid) the warning and just leave DeformationFieldFileName as an empty string instead?  https://github.com/SuperElastix/elastix/blob/5.0.1/Components/Transforms/DeformationFieldTransform/elxDeformationFieldTransform.hxx#L180-L182

Reply:
We could assume empty string for lastpart, when TransformParameterFileName  is empty.